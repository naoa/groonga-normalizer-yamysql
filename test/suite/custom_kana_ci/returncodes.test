register normalizers/yamysql

normalize NormalizerYaMySQLKanaCI 'a\nb' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLKanaCI 'a\rb' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLKanaCI 'a\r\nb' WITH_TYPES|WITH_CHECKS

normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml 'a\nb' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml 'a\rb' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml 'a\r\nb' WITH_TYPES|WITH_CHECKS

normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml 'a\n' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml 'a\r' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml 'a\r\n' WITH_TYPES|WITH_CHECKS

normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml '\nb' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml '\rb' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml '\r\nb' WITH_TYPES|WITH_CHECKS

normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml 'a*b' WITH_TYPES|WITH_CHECKS
