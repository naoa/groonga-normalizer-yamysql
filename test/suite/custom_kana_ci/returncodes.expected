register normalizers/yamysql
[[0,0.0,0.0],true]
normalize NormalizerYaMySQLKanaCI 'a\nb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"AB","types":["alpha","alpha"],"checks":[1,2]}]
normalize NormalizerYaMySQLKanaCI 'a\rb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"AB","types":["alpha","alpha"],"checks":[1,2]}]
normalize NormalizerYaMySQLKanaCI 'a\r\nb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"AB","types":["alpha","alpha"],"checks":[1,3]}]
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml 'a\nb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"AB","types":["alpha","alpha"],"checks":[1,2]}]
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml 'a\rb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"AB","types":["alpha","alpha"],"checks":[1,2]}]
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml 'a\r\nb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"AB","types":["alpha","alpha"],"checks":[1,3]}]
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml 'a\n' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"A","types":["alpha"],"checks":[1]}]
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml 'a\r' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"A","types":["alpha"],"checks":[1]}]
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml 'a\r\n' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"A","types":["alpha"],"checks":[1]}]
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml '\nb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"B","types":["alpha"],"checks":[2]}]
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml '\rb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"B","types":["alpha"],"checks":[2]}]
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml '\r\nb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"B","types":["alpha"],"checks":[3]}]
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml 'a*b' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"AB","types":["unknown","alpha"],"checks":[1,2]}]
